packages/adapter-cloudflare-workers/index.js

```diff
@@ -67,7 +67,7 @@ export default function () {
 	};
 }
 
-/** @param {import('@sveltejs/kit').Builder} builder */
+/** @param {any} builder */
```

Comment:
this is a bit cheeky but i'm not sure if it's worth making `Builder` public

Reply:
The only thing we gain by not making it public is that we can rename it from `Builder` to something else, but we can't change any of the methods on it in any manner, so the API is already 95% locked. Personally, I would be in favor of making it public

Reply:
true. updated